public/scripts/power-user.js

```diff
@@ -1747,7 +1747,7 @@ async function loadContextSettings() {
         // If the setting already exists, no need to duplicate it
         // TODO: Maybe check the power_user object for the setting instead of a flag?
         $element.on('input keyup', async function () {
-            const value = control.isCheckbox ? !!$(this).prop('checked') : $(this)[0].innerText;
+            const value = control.isCheckbox ? !!$(this).prop('checked') : getTextInputContent($(this)[0]);
```

Comment:
May be prudent to debounce this given that `getTextInputContent` is not exactly lightweight -- my intent was more that it would be called only on save, not on every keydown.